### ISSUE TITLE
[VT-20] vit; bind "q" to prompted quit

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#42]
_David Patrick on 2013-05-15T21:14:21Z says:_ 

 I've trained myself not to hit just "q" when quitting vit, but it took a while and I still do it.
Why not bind "q" to a prompted [ quit ? Y/n ] ? And then why not bind "Q" to an umprompted quit (like ZZ) ? 
It's not like those keybindings are going to make sense for anything other than quitting, and it will mean far fewer instances of us going DOH!!

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-05-15T21:14:21Z
Modified: 2014-02-09T01:16:50Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-08-13T01:35:52Z says:_ 

 Fixed at 2efb4967. Can you test that it does what you had in mind?

> It's not like those keybindings are going to make sense for anything other than quitting

I don't necessarily agree. As I note in the commit message:

<pre>
    Note that this could change in the future because it is not
    Vim-like. In Vim, 'q' starts a macro. If in Vit something
    analogous is implemented, the 'q' key might be taken.
</pre>

Note that if a user does not like these bindings, they can be disabled by shortcuts (which just need to be put in ~/.vimrc):
<pre>
map q=
map Q=
</pre>

**Comment by taskwarrior** [comment]
_David Patrick on 2013-08-13T02:15:55Z says:_ 

 Scott Kostyshak wrote:
> Fixed at 2efb4967. Can you test that it does what you had in mind?

will do!
> 
> > It's not like those keybindings are going to make sense for anything other than quitting
> 
> I don't necessarily agree. As I note in the commit message:

I hadn't considered starting a macro! Let's see how it's behaving, thanks!

> Note that if a user does not like these bindings, they can be disabled by shortcuts (which just need to be put in ~/.vimrc):
> [...]

that's just brilliant!
